src/pybind/mgr/ansible/module.py

```diff
@@ -382,7 +396,7 @@ def get_inventory(self, node_filter=None, refresh=False):
 
         # Assign the process_output function
         ansible_operation.process_output = process_inventory_json
-        ansible_operation.event_filter = "runner_on_ok"
+        ansible_operation.event_filter = ["runner_on_ok"]
```

Comment:
This is a minor thing but, if you already initialized `self.event_filter` to a list, by doing this you're discarding that initial list object and creating a new one. Why not appending to the exiting list (or encapsulating)?
``` python
 ansible_operation.event_filter += ["runner_on_ok"]
 # or
 ansible_operation.event_filter.append("runner_on_ok")
```

Reply:
I'd favor `=` here, as this is more like an additional, optional parameter

Reply:
I changed the name of the variable.... i really want here a list of possible events to use as "filter" criteria. The list is assigned each time a new AnsibleOperation is configured.